doc/api/events.md

```diff
@@ -1247,12 +1247,21 @@ target.addEventListener('foo', handler4, { once: true });
 ### `EventTarget` error handling
 
 When a registered event listener throws (or returns a Promise that rejects),
-by default the error is forwarded to the `process.on('error')` event
-on `process.nextTick()`. Throwing within an event listener will *not* stop
-the other registered handlers from being invoked.
+by default the error is treated as an uncaught exception on
+`process.nextTick()`. This means uncaught exceptions in `EventTarget`s will
+crash the Node.js process by default.
 
-The `EventTarget` does not implement any special default handling for
-`'error'` type events.
+Throwing within an event listener will *not* stop the other registered handlers
+from being invoked.
+
+The `EventTarget` does not implement any special default handling for `'error'`
+type events like `EventEmitter` does in order to be spec compliant.
```

Comment:
```suggestion
type events like `EventEmitter` does to be spec compliant.
```

...or even:

```suggestion
type events like `EventEmitter` does.
```

It's not clear without additional context which is being spec compliant. Is it `EventTarget` being spec compliant but not implementing it? Or is it `EventEmitter` being spec compliant by implementing it? (I know the answer, or at least I think I do, but a reader may not.) Then again, maybe it's not even relevant, hence the removal suggestion above.